docs/i18n-languages/portuguese/README.md

```diff
@@ -130,7 +130,7 @@ A nossa comunidade também tem:
 
 ### Reportar Bugs e Issues
 
-Se pensas que encontraste um bug, primeiro lê o article [como reportar um bug](https://www.freecodecamp.org/forum/t/how-to-report-a-bug/19543) e segue as suas instruções.
+Se pensas que encontraste um bug, primeiro lê o artigo [como reportar um bug](https://www.freecodecamp.org/forum/t/how-to-report-a-bug/19543) e segue as suas instruções.
 
 Se estás confiante que é um novo bug, e confirmaste que outra pessoa se está a deparar com o mesmo problema, segue em frente e cria um novo GitHub issue. Assegura que incluis o máximo de informação possível para que consigamos replicar o erro.
```

Comment:
> segue em frente e cria um novo GitHub issue.

Wouldn't it be better as "cria um novo Issue no GitHub"? Not sure if this would be the correct way to mention it, but quoting "GitHub issue" sounds kind of __not translated__, maybe should we add quotation marks around these words?

Reply:
You're right! I've even used that translation below. This is fixed on this latest commit.